plumbing/cache/object_lru.go

```diff
@@ -55,18 +55,14 @@ func (c *ObjectLRU) Put(obj plumbing.EncodedObject) {
 		return
 	}
 
-	if c.actualSize+objSize > c.MaxSize {
+	for c.actualSize+objSize > c.MaxSize {
```

Comment:
Should we check if the object to add is bigger than the cache size to do not remove everithing?

Reply:
nevermind, is already checked before the for.